src/components/ViewComments.jsx

```diff
@@ -32,21 +52,29 @@ const ViewComments = () => {
             comment_id: Date.now()
         };
         postComment(article_id, newComment).then((myComment) => {
-            setErr(null)
+            setPostErr(null)
             setComments((currComments) => {
                 return [tempComment, ...currComments]
             })
         }).catch((err)=> {
             setComments((currComments) => currComments = "");
-            setErr(true)
+            setPostErr(true)
         })
         
     }
 
+    if (deleteerr) return <h4>'We are currently not allowing any take backs - please refresh the page and try again'</h4> 
+
+    if (isDeleting) return <p>Comment deleting....</p>
+
+    if (deleted) return <p>We don't usually allow take backs but we can make an exception. Your comment is deleted <br />
```

Comment:
I like this way of confirming the comment has been deleted and as always a great choice of message back to the user 😄